OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -45,6 +45,9 @@ void SwitchMenu(MenuType type)
 		{
 			menuType = type;
 
+			if (type == MenuType.Main)
```

Comment:
This unfortunately doesn't work right thanks to the way we return to the menu from a lobby / game.
* When returning from a skirmish/mp lobby we close the window and drop back to the previous (e.g. Singleplayer) menu without triggering this - the status therefore doesn't update back to the menu
* When returning from a skirmish game we briefly go via the main menu before returning to a new skirmish lobby - we therefore show an incorrect menu status for a second or two.

We can solve both issues by using `ChromeLogic.Tick` to check the current status and set a new one if needed. This would be much cleaner if we used an enum to represent the different states instead of freeform labels.